specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -1329,6 +1329,72 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/{integrationRuntimeAction}/operationstatuses/{integrationRuntimeOperationId}": {
```

Comment:
>{integrationRuntimeAction} [](http://example.com/codeflow?start=172&length=26)

This doesn't seem right. The "integrationRuntimeAction" should form a valid resource in the operation URI, in even number of tuples, similar to "/integrationRuntimes/{integrationRuntimeName}". Only identifier is not valid.